modules/authorization-xacml/src/main/java/org/opencastproject/authorization/xacml/XACMLAuthorizationService.java

```diff
@@ -202,14 +203,10 @@
         // add attachment
         final String elementId = toElementId(scope);
         URI uri;
-        InputStream in = null;
-        try {
-          in = IOUtils.toInputStream(xacmlContent, "UTF-8");
+        try (InputStream in = IOUtils.toInputStream(xacmlContent, "UTF-8")) {
           uri = workspace.put(mp.getIdentifier().toString(), elementId, XACML_FILENAME, in);
         } catch (IOException e) {
           throw new MediaPackageException("Error storing xacml for mediapackage " + mp.getIdentifier());
-        } finally {
-          IOUtils.closeQuietly(in);
```

Comment:
`IOUtils.closeQuietly(…)` is deprecated and we can simply use a try-with-resources statement